detections/endpoint/windows_bidirectional_communication_via_telegram_bot.yml

```diff
@@ -0,0 +1,67 @@
+name: Windows Bidirectional Communication via Telegram Bot
```

Comment:
I know of the convention but the title is confusing and does not reflect the logic. The rule is looking for initiated DNS query request. So it should be something that reflect that.

Reply:
fixed!